SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -344,6 +344,9 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(Config)
 	bool bUseSpatialView;
 
+	UPROPERTY(EditAnywhere, config, Category = "Connection")
+	bool bSpatialNetworking;
```

Comment:
Set a default value to this parameter if Unreal doesn't load a value from INI file.

Reply:
Assign the default true value in the initializing list